src/gui/mainwindow.cpp

```diff
@@ -326,6 +326,15 @@ MainWindow::MainWindow(QWidget *parent)
     m_ui->actionCheckForUpdates->setVisible(false);
 #endif
 
+#ifdef Q_OS_MAC
```

Comment:
Hmm, are you sure that will not cause undesired effects in the future if Qt decides to change their policy or something? At the moment roles only affects macs and there is no real need for any other platform to have them. 

Also I cannot test on other platforms, so I would rather not add mac-specific functionality everywhere right before a release.